app/ios/Podfile.lock

```diff
@@ -269,18 +269,18 @@ PODS:
     - React-jsi (= 0.65.1)
```

Comment:
As this PR isn't about making any package changes, I don't think you should need to include this file?

Reply:
This is no longer included.